ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/ExceptionRequestModal/CompletedExceptionRequestModal.tsx

```diff
@@ -25,7 +85,46 @@ function CompletedExceptionRequestModal({
                 </Button>,
             ]}
         >
-            {exceptionRequest.id}
+            <Flex direction={{ default: 'column' }}>
+                <Text>Use this link to share and discuss your request with your approver.</Text>
+                <ClipboardCopy
+                    isReadOnly
+                    hoverTip="Copy"
+                    clickTip="Copied"
+                >{`todo.path.to.requests.page/${exceptionRequest.name}`}</ClipboardCopy>
```

Comment:
Since this is a growing capability, is this a repeatable pattern for the start of a page address?
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/MainPage/InviteUsers/InviteUsersConfirmationNoEmail.tsx#L94

Reply:
Definitely, I think so. It may be worth creating a global util for this, or seeing if something is available for export in react router.

Reply:
Aha, an external full-path counterpart to internal relative-path from `generatePath` function that we have started using recently?